tests/conftest.py

```diff
@@ -30,6 +30,7 @@ def oauth_target():
         'method': 'oauth',
         'threads': 1,
         # project isn't needed if you configure a default, via 'gcloud config set project'
+        'location': 'US'
```

Comment:
I don't think it's bets to have location as a default in the profile but I was in a crunch to try to get the tests working.  We should add location for these tests specifically and probably add a test for BQ to make sure the location is in the profile when doing grants.